### PR TITLE
Translate several version notes into Traditional Chinese (zh_tw)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

File: zh_tw/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+layout: news_post
+title: "CVE-2017-17405: Net::FTP 命令注入安全性風險"
+author: "nagachika"
+translator: "Delton Ding"
+date: 2017-12-14 16:00:00 +0000
+tags: security
+lang: zh_tw
+---
+
+Ruby 內建的 Net::FTP 存在一個命令注入風險。此風險的 CVE 識別號為  [CVE-2017-17405](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-17405)。
```
Comment (bugtender) [inline zh_tw/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md]: There is two space between `識別號為` and `[CVE-2017-17405]`.

File: zh_tw/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* Ruby 2.4 系列： 2.4.2 及之前版本
+* Ruby 2.5 系列： 2.5.0-preview1
+* trunk revision 早於 r61242 的全部版本
+
+## 致謝
+
+感謝 Heroku 產品安全團隊的 Etienne Stalmans 回報了這一問題。
+
+## 歷史
+* 于 2017-12-14 16:00:00 (UTC) 首次發佈
```
Comment (bugtender) [inline zh_tw/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md]: `於` is better than `于`  in zh_tw.

File: zh_tw/news/_posts/2017-12-14-ruby-2-4-3-released.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+lang: zh_tw
+---
+
+Ruby 2.4.3 已發佈
+
+此次發佈修復了多個漏洞，包括一個安全性修正。
+
+* [CVE-2017-17405: Net::FTP 命令注入安全性風險](/zh_tw/news/2017/12/14/net-ftp-command-injection-cve-2017-17405/)
+
+其他問題請查看 [提交记录](https://github.com/ruby/ruby/compare/v2_4_2...v2_4_3) 以獲取詳細訊息。
```
Comment (bugtender) [inline zh_tw/news/_posts/2017-12-14-ruby-2-4-3-released.md]: There is `提交紀錄` in zh_tw.

File: zh_tw/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+date: 2017-12-14 16:00:00 +0000
+tags: security
+lang: zh_tw
+---
+
+Ruby 內建的 Net::FTP 存在一個命令注入風險。此風險的 CVE 識別號為  [CVE-2017-17405](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-17405)。
+
+## 細節
+
+`Net::FTP#get`、`getbinaryfile`、`gettextfile`、`put`、`putbinaryfile` 和 `puttextfile` 使用 `Kernel#open` 來打開本地檔案。如果 `localfile` 命令以 pipeline 字符 `"|"` 開始，此後的字符會以命令被執行。`localfile` 的缺省值是 `File.basename(remotefile)`，那麼惡意 FTP 伺服器可能引發任意代碼的執行。
```
Comment (bugtender) [inline zh_tw/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md]: I think `預設值` is better. 🤔

File: zh_tw/news/_posts/2017-12-14-ruby-2-2-9-released.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+date: 2017-12-14 16:00:00 +0000
+lang: zh_tw
+---
+
+Ruby 2.2.9 已發佈。此次發佈包括數個安全性修正。請查看以下文章以獲取詳細訊息。
+
+* [CVE-2017-17405: Net::FTP 命令注入安全性風險](/zh_tw/news/2017/12/14/net-ftp-command-injection-cve-2017-17405/)
+* [RubyGems 中不安全物件反序列化風險](http://blog.rubygems.org/2017/10/09/unsafe-object-deserialization-vulnerability.html)
+
+Ruby 2.2 處於安全維護期，于 2018 年 3 月停止。此後 Ruby 2.2 的維護會全部停止。我們建議您著手遷移到更新的 Ruby 版本，例如 2.4 或 2.3。
```
Comment (bugtender) [inline zh_tw/news/_posts/2017-12-14-ruby-2-2-9-released.md]: `於` is better than `于` in zh_tw.

**Comment by dsh0416** [comment]
Problems mentioned have been fixed.

**Pull request merged**